apps/storybook/src/Selection.stories.tsx

```diff
@@ -67,7 +66,13 @@ const Template: Story<TemplateProps> = (args) => {
         onSelectionEnd={() => setActiveSelection(undefined)}
         modifierKey={selectionModifierKey}
       >
-        {(selection) => <SelectionComponent {...selection} {...svgProps} />}
+        {({ data: [dataStart, dataEnd] }) => (
+          <SelectionComponent
+            startPoint={dataStart}
+            endPoint={dataEnd}
```

Comment:
I'm hoping I can refactor this in my next PR to avoid the array destructuring. Also, we have access to the world selection, so we could avoid the `dataToWorld` conversion in `Selection<Rect|Line>`.